src/components/App.js

```diff
@@ -1,4 +1,6 @@
 import React, { Component } from 'react';
+import ClassNames from 'classnames';
```

Comment:
This is named `classNames` in all other files -- follow that pattern.

Reply:
Done